eng/packaging.props

```diff
@@ -20,7 +20,7 @@
     <SkipBaseLineCheck>true</SkipBaseLineCheck>
 
     <!-- by default all packages will use the same version which revs with respect to product version -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">5.0.0</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">6.0.0</PackageVersion>
```

Comment:
Not for this PR, but we should try to make this use $(ProductVersion) if possible.